FirebaseCore/Sources/FIRApp.m

```diff
@@ -32,7 +32,9 @@
 
 #import "FirebaseCore/Extension/FIRAppInternal.h"
 #import "FirebaseCore/Extension/FIRCoreDiagnosticsConnector.h"
+#ifndef FIREBASE_BUILD_CMAKE
 #import "FirebaseCore/Extension/FIRHeartbeatLogger.h"
```

Comment:
Hmm since everything in the `FIRHeartbeatLogger` header is ifdef'ed out already, is this necessary?

Reply:
Oh darn, good catch. I did it this way originally. Thanks for thorough review!!

Reply:
Done in [42144ea](https://github.com/firebase/firebase-ios-sdk/pull/9537/commits/42144ea844d345a3b5ca3c02591ce66dcb393903)

Reply:
I'll wait for CMake CI to go green before merging.